README.md

```diff
@@ -139,7 +139,7 @@ Create the `vault` Kubernetes cluster:
 ```
 gcloud container clusters create vault \
   --enable-autorepair \
-  --cluster-version 1.11.2-gke.9 \
+  --cluster-version 1.11.2-gke.15 \
```

Comment:
We should just change this to 1.11.2 or even 1.11. gcloud will do the right thing and then we wont have to keep updating it 😄